src/Validation/validateInput.js

```diff
@@ -0,0 +1,10 @@
+export default function validateInput(inputMail, inputPassword) {
+  var mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+  if (inputMail == '' || mailformat.test(inputMail) == false) {
+    alert('Please provide valid email');
```

Comment:
You are using the alert() function multiple times. Although, it's just a single-line code, but we can create a separate function for it let say showAlert(title) which will take some arguments which we need to use inside alert().